### DISCUSSION TITLE
is there any way to hide this menu ?

### DISCUSSION THREAD

**Discussion by Aditya8840** [excalidraw/excalidraw#8930] [General]
<img width="236" alt="Screenshot 2024-12-20 at 12 56 46 AM" src="https://github.com/user-attachments/assets/4ce63181-e6c5-4d1a-bda0-c6b91ac3d65b" />

The placement of the menu in Excalidraw's shared window interfered with my ability to use the canvas effectively.

**Comment by Mrazator** [comment]
Have you tried "Zen mode"?

<img width="282" alt="Screenshot 2024-12-23 at 19 17 29" src="https://github.com/user-attachments/assets/a9cc3e64-c4c5-4b6e-8f6a-4ba402b76ffc" />

**Reply by Aditya8840** [reply]
Ohk, Thanks !
